marketplace-builder/src/builder.rs

```diff
@@ -526,6 +526,8 @@ mod test {
         parent_view_number: u64,
         parent_commitment: VidCommitment,
     ) -> Bundle<SeqTypes> {
+        // FIXME error here:
```

Comment:
link

Reply:
it is coming from here https://github.com/EspressoSystems/HotShot/blob/main/crates/builder-api/src/v0_3/mod.rs#L6

https://github.com/EspressoSystems/HotShot/blob/main/crates/builder-api/src/v0_3/builder.rs#L12C31-L12C38